src/src/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -306,4 +323,14 @@ void setTenantId(String tenantid) {
     void setRefreshToken(String refreshToken){
         mRefreshToken = refreshToken;
     }
+    
+    final String getFamilyClientId()
+    {
+        return mFamilyClientId;
+    }
+    
+    final void setFamilyClientId (final String familyClientId)
```

Comment:
And why is it so? It might be useful for UnitTesting to be able to override these methods